src/github.com/getlantern/geolookup/geolookup_test.go

```diff
@@ -0,0 +1,47 @@
+package geolookup
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestCityLookup(t *testing.T) {
+	var city *City
+	var err error
+
+	if city, err = LookupCity("198.199.72.101"); err != nil {
+		t.Fatal(err)
+	}
+
+	if city.City.Names["en"] != "New York" {
+		t.Fatal("Look up failed.")
+	}
+
+}
+
+func TestUsingDefaultClient(t *testing.T) {
+	if UsesDefaultHTTPClient() == false {
```

Comment:
I'd recommend using github.com/getlantern/testify/assert because it makes test conditions like this easier to read.